.github/workflows/update_sdk.yml

```
@@ -0,0 +1,132 @@
+name: Update SDK
+
+on:
+  schedule:
+    - cron: "0 * * * *" # Run each hour
+  workflow_dispatch:
+
+jobs:
+  update-sdk:
+    name: "Update ${{ matrix.sdk.commit_message }}"
+    runs-on: windows-latest
+
+    strategy:
+      matrix:
+        sdk:
+          - name: "Microsoft.WindowsAppSDK"
+            commit_message: "Windows App SDK"
+            allow_prerelease: false
+            script_file_name: "scripts\\get_appsdk.ps1"
+            json_file_name: "WindowsAppSDK.json"
+          - name: "Microsoft.Web.WebView2"
+            commit_message: "Microsoft.Web.WebView2"
+            allow_prerelease: false
+            script_file_name: "scripts\\get_appsdk.ps1"
+            json_file_name: "WindowsAppSDK.json"
+          - name: "Microsoft.Windows.SDK.Win32Metadata"
+            commit_message: "Win32Metadata"
+            allow_prerelease: true
+            script_file_name: "scripts\\get_win32.ps1"
+            json_file_name: "Windows.Win32.json"
+          - name: "Microsoft.Windows.SDK.Contracts"
+            commit_message: "Windows SDK"
+            allow_prerelease: true
```

```suggestion
            allow_prerelease: false
```
IMO: because it has stable release.

---

Fixed in https://github.com/ynkdir/py-win32more/pull/66/commits/9060873a3c71a4931978d833c4975c1f307c8dbf